ldap/servers/slapd/ldaputil.c

```diff
@@ -508,17 +508,13 @@ setup_ol_tls_conn(LDAP *ld, int clientauth)
     int ssl_strength = 0;
     int rc = 0;
     const char *cacert = NULL;
+    char **cauris = NULL;
     char *errmsg = NULL;
 
-    /* certdir is used to setup outgoing secure connection (openldap)
-     * It refers to the place where PEM files have been extracted
-     *
-     * If a private tmp namespace exists
-     * it is the place where PEM files have been extracted
-     */
-    if ((certdir = check_private_certdir()) == NULL) {
-        certdir = config_get_certdir();
-    }
+    /* if set, cacerturis takes precedence */
+#ifdef LDAP_OPT_X_TLS_CACERTURIS
+    ldap_get_option(ld, LDAP_OPT_X_TLS_CACERTURIS, &cauris);
```

Comment:
At this point, is it possible that LDAP_OPT_X_TLS_CACERTURIS is set ?
IIUC setup_ol_tls_conn is called before LDAP_OPT_X_TLS_CACERTURIS is set (in conn_connect_with_bootstrap)

Reply:
This was working by accident I think, it was't set on the first pass - thanks for catching this.

Fixed below.